src/components/PrintModal/PrintModal.js

```diff
@@ -393,7 +394,7 @@ class PrintModal extends React.PureComponent {
     printHandler.appendChild(fragment);
 
     // Print for Safari browser. Makes Safari 11 consistently work.
-    if (!document.execCommand('print')) {
+    if (!document.execCommand('print') || isChromeOniOS) {
```

Comment:
Hmm I think at first we use `window.print` for every browser and then we found we need to use `document.execCommand('print')` or Safari...

Do you think writing in this way is a bit cleaner?
```javascript
if (isSafari) {
  // Print for Safari browser. Makes Safari 11 consistently work.
  document.execCommand('print');
} else {
  window.print();
}
```

Reply:
I can update this, but it looked like we are using  "document.execCommand('print') " as the default and fallback `to window.print()` if it fails. We are kind of changing the logic in your snippet ( but I guess if we know `document.execCommand('print')` will fail for everything beside Safari, it fine)

Reply:
Yeah my snippet changed the logic but since at the very beginning we only do `window.print()`, I think we can just use this for all the browsers other than Safari. Not sure if I'm making sense.

Reply:
I updated the code